src/shared/utilities/promptSetting.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { getLogger } from '../logger'
+import { SettingsConfiguration } from '../settingsConfiguration'
+
+export type PROMPT_NAME =
+    | 'warnBeforeRunCommand'
+    | 'warnBeforeEnablingExcecuteCommand'
+    | 'warnBeforeDisablingExecuteCommand'
+export const DEFAULT_PROMPTS: PromptSetting = {
+    warnBeforeRunCommand: true,
+    warnBeforeEnablingExcecuteCommand: true,
+    warnBeforeDisablingExecuteCommand: true,
```

Comment:
This is not needed unless we plan to introduce a prompt that defaults to `false`--which would be very unlikely, and is something we should wait for rather than writing code to anticipate it.